src/api/nodeRpcCall.js

```diff
@@ -5,9 +5,8 @@ import { USER_ROLES, userRolesHelper } from '../utils/userRolesHelper';
 import { handleMyDispatchErrors } from '../utils/therapist';
 import { blockchainDataToFormObject } from '../utils/registryFormBuilder';
 import * as centralizedBackend from './backend';
-import {
-  convertAssetData,
-} from '../utils/dexFormater';
+// eslint-disable-next-line import/no-cycle
```

Comment:
It's because in dexFormatter we have import from nodeRpcCall, and in nodeRpcCall i have import from dexFormatter? should i just move this function convertAssetData to nodeRpcCall?

Reply:
In perfect world I'd keep `nodeRpcCall` as a very thin wrapper over actual APIs and not put business logic in here at all. `convertAssetData` is something that fits more in sagas.

But I don't care much, there's a lot of business logic in here either way. I just missed the import of `nodeRpcCall` in `dexFormatter` and was genuinely confused as to why we need to silence eslint here. If it's a real cycle and it causes no issues I'm fine leaving it as-is.